### DISCUSSION TITLE
Downloading bathymetry data into GeoPackage

### DISCUSSION THREAD

**Discussion by EMaksymenko** [ngageoint/geopackage-mapcache-android#38] [Ideas]
Do you plan to add possibility to download bathymetry data like GEBCO or SRTM into GeoPackage TIFF or bil16 format?

Technically it is also a raster image tiles but with height values instead of map pixels.

What problem do you see with storing elevation data in GeoPackage?

**Comment by bosborn** [comment]
The [android](https://github.com/ngageoint/geopackage-android) and [android-map](https://github.com/ngageoint/geopackage-android-map) libraries currently support elevations through the [OGC GeoPackage Extension for Tiled Gridded Coverage Data](http://docs.opengeospatial.org/is/17-066r1/17-066r1.html).

However, this app does not currently visualize those elevations.

**Comment by EMaksymenko** [comment]
@bosborn Hi. Is there any gpkg file example with Tiled Gridded Coverage Data exists or how to create it with your Android application to be able to analyze a file content?

I am searching some example to understand how SQLite file looks like inside to integrate result of library with WorldWind GIS, because official documentation is not clear for me.

**Comment by bosborn** [comment]
[This example GeoPackage](https://github.com/ngageoint/GeoPackage/tree/master/docs/examples/android) has both a PNG and TIFF version of the extension.  It is just fake data used to test and pass certification, but should help understanding the SQLite file setup.

**Comment by EMaksymenko** [comment]
Thanks. This is what I have searched for.

By the way, I am now developing Kotlin Multi-platform (Java, Android, JS) version of WorldWind GIS based on ideology of separate NASA WorldWind repositories. Exactly now I am implementing Elevation cache based on GeoPackage format. May be you will be interested on using it instead of Google Maps in your examples.
